errors.txt

```diff
@@ -0,0 +1,9 @@
+Empty text file:
```

Comment:
is this a log file? usually they have the .log extension

Reply:
Nope, file to keep track of the different errors I have been able to produce while opening a dicom file so we can try and be more specific on what is missing etc.